noaa.go

```diff
@@ -175,13 +232,15 @@ type GridpointForecastResponse struct {
 	Point                            *PointsResponse
 }
 
+type GridpointForecastTimeSeriesValue struct {
+	ValidTime string  `json:"validTime"` // ISO 8601 time interval, e.g. 2019-07-04T18:00:00+00:00/PT3H
+	Value     float64 `json:"value"`
+}
+
 // GridpointForecastTimeSeries holds a series of data from a gridpoint forecast
 type GridpointForecastTimeSeries struct {
-	Uom    string `json:"uom"` // Unit of Measure
-	Values []struct {
-		ValidTime string  `json:"validTime"` // ISO 8601 time interval, e.g. 2019-07-04T18:00:00+00:00/PT3H
-		Value     float64 `json:"value"`
-	} `json:"values"`
+	Uom    string                              `json:"uom"` // Unit of Measure
+	Values []*GridpointForecastTimeSeriesValue `json:"values"`
```

Comment:
Same, make this a slice of structs instead of pointers.

Reply:
Ack.